source/_posts/2019-10-20-release-101.markdown

```diff
@@ -57,24 +57,56 @@ Experiencing issues introduced by this release? Please report them in our [issue
 
 ## Breaking Changes
 
-- Add turn_on method to ecobee climate platform ([@marthoc] - [#27103]) ([ecobee docs]) (breaking change)
-- UniFi - Bandwidth sensors ([@Kane610] - [#27229]) ([unifi docs]) (breaking change) (new-platform)
-- Refactor IKEA Tradfri, part 2 ([@ggravlingen] - [#27245]) ([tradfri docs]) (breaking change)
-- Remove hipchat ([@fabaff] - [#27399]) ([hipchat docs]) (breaking change)
-- Standardize times in time sensors Jewish calendar ([@tsvi] - [#26940]) ([jewish_calendar docs]) (breaking change)
-- Fix pioneer volume when using onkyo component ([@foxy82] - [#27218]) ([onkyo docs]) (breaking change)
-- Add abode config entries and device registry ([@shred86] - [#26699]) ([abode docs]) (breaking change)
-- Allow MQTT json light floating point transition ([@starkillerOG] - [#27253]) ([mqtt docs]) (breaking change)
-- Remove direct authentication via trusted networks or API password ([@balloob] - [#27656]) ([auth docs]) ([http docs]) ([websocket_api docs]) (breaking change)
-- Deprecate Python 3.6 support, 3.8.0 is out ([@scop] - [#27680]) (breaking change)
-- Move holiday info into a single sensor with multiple attributess ([@tsvi] - [#27654]) ([jewish_calendar docs]) (breaking change)
-- Introduce new OAuth2 config flow helper ([@balloob] - [#27727]) ([somfy docs]) (breaking change)
-- Add support for AdGuard Home v0.99.0 ([@frenck] - [#27926]) ([adguard docs]) (breaking change)
-- remove duplicate unique_id, add unique_id for issues ([@zxdavb] - [#27916]) ([geniushub docs]) (breaking change)
-- Remove stride ([@hmmbob] - [#27934]) ([stride docs]) (breaking change)
-- Glances config flow ([@engrbm87] - [#27221]) ([glances docs]) (breaking change)
-- Bump teslajsonpy and add update switch ([@alandtse] - [#27957]) ([tesla docs]) (breaking change)
-- changed STATE_OFF to STATE_STANDBY ([@Villhellm] - [#28148]) ([roku docs]) (breaking change)
+- __Removed__ - The following integrations have been removed:
+  - __Direct authentication via trusted networks or API password__ - It is no longer possible to make authenticated requests using trusted networks or by appending ?api_password=X to the url. You will now first need to get an authentication token and use that token to make requests. - ([@balloob] - [#27656]) ([auth docs]) ([http docs]) ([websocket_api docs])
+
+  - __Stride__ - The Stride notification integration was discontinued in February 2019 due to a take over by Slack. - ([@hmmbob] - [#27934]) ([stride docs])
+  - __Hipchat__ - The Hipchat notification integration was discontinued in February 2019, also due to a take over by Slack. - ([@fabaff] - [#27399]) ([hipchat docs])
+- __Deprecated__ - Python 3.6 support is deprecated, as version 3.8.0 is now out -  ([@scop] - [#27680])
```

Comment:
Should we mention the date that support is removed?  I think its december 15?

Reply:
Thanks. Info added.